docs/src/components/features/panel/panel.html

```diff
@@ -1,37 +1,40 @@
-<section class="panel">
-    <header class="panel-header row-center-x">
+<!-- `div` should be `section` -->
```

Comment:
Brought this up to date w production code, but made a note where we should update production code semantics